src/pepr/operator/crd/sources/package/v1alpha1.ts

```diff
@@ -299,8 +299,32 @@ const sso = {
       attributes: {
         description: "Specifies attributes for the client.",
         type: "object",
-        additionalProperties: {
-          type: "string",
+        properties: {
+          "oidc.ciba.grant.enabled": {
+            type: "string",
+          },
+          "backchannel.logout.session.required": {
+            type: "string",
+          },
+          "backchannel.logout.revoke.offline.tokens": {
+            type: "string",
+          },
+          "post.logout.redirect.uris": {
+            type: "string",
+          },
```

Comment:
Maybe just for my understanding - how does this differ from our current `redirectUris` in the sso spec/do we know of places this is being used?

Reply:
The existing `redirectUris` is used for post *login* redirect, the attributes is used for post *logout* redirects. I don't believe any of our current clients are utilizing this fields outside of the default `*` but could be useful for certain applications that don't have the redirect configuration.

Reply:
Ah that's helpful. I think okay adding this here, no real risk exposing it and seems like it could be useful.